debug.go

```diff
@@ -107,11 +110,34 @@ const pageHTML = `
 			border-color: #3A3A3A;
 			background-color: #ffffff;
 		}
+		header::after {
```

Comment:
ah gotcha. This is the only file containing an html template for now. I will move it to its own whenever I start working on the in-flight request tracing. As this will require more templates. Everything will likely move to a sub-package.